content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+
+﻿*The above figure shows the guide for protection-policies request body JSON structure*
+
+To simplify this example, I created a three-tier protection-policy for VMware as depicted by this snippet from the protection policy’s menu.
+
+![3 Tiers protection policy ](/img/backup-protection-policy-with-3-tiers.png)
+
+The list of the steps to create this protection policy:
+
+1. From the figure below, inside the protection store gateways menu, I discovered the serial number of the protection-store gateway and used it as the filtering parameter to obtain the protection store gateway instance.  
+
+![PSG UI](/img/psg-ui-to-show-the-serial-no.png)
+
+2. Afterward, I used `GET /backup-recovery/v1beta1/protection-store-gateways` [API](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/openapi/backup-recovery-public-v1beta1/operation/ProtectionStoreGatewaysList) to figure out the `"<protection-store-gateway-id>"` that was associated with the protection-stores that would be incorporated into the protection-policy.
+
+![API show registered PSG](/img/api-display-registered-psg.png)
+
+3. From the figure below, I used `GET /backup-recovery/v1beta1/protection-stores` [API](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/openapi/backup-recovery-public-v1beta1/operation/ProtectionStoreList/) to obtain the protection-store ids for both the on-premises protection store and the cloud protection store. To display protection-stores related to the protection store gateway, I used the parameter filter to display the exact the protection-store associated with protection storage gateway of `“<onprem-protection-store-id>”`. The filter parameter that I used are `protectionStoreType eq 'ON_PREMISES'` and `storageSystemInfo/id eq 'protection-store-gateway-id'`. Additionally, I used the following `select` parameter `name,displayName,id,status,state,protectionStoreType` to provide shorter response that simplify the discovery of the protection-store on-premises. The API used for this: 
+
+   ```shellsession
+   GET /backup-recovery/v1beta1/protection-stores?select=name,displayName,id,status,state,protectionStoreType&filter=protectionStoreType eq ‘ON_PREMISES’ and storageSystemInfo/id eq “\<protection-store-gateway-id\>”
+   ```
+
+![API to obtain the onpremises protection store id](/img/api-to-get-onpremises-protection-store-id.png)
+
+4. I repeat the same execution of the `GET /backup-recovery/v1beta1/protection-stores` to obtain the **“<cloud-protection-store-id>”**. To accomplish that, I used the following `filter: protectionStoreType eq 'CLOUD' and storageSystemInfo/id eq “<protection-store-gateway-id>”`.  Additionally, I also used the  parameter `select: name,displayName,id,status,state,protectionStoreType` to provide shorter response for simpler discovery of the protection-store-id in the cloud. The API used for this: 
```

Comment:
"I repeat**ed**" [past tense]

Again, later on in the paragraph "to provide **a** shorter response..."